rust/log/src/sqlite_log.rs

```diff
@@ -378,6 +386,37 @@ impl SqliteLog {
 
         Ok(())
     }
+
+    pub async fn purge_logs(
+        &mut self,
+        collection_id: CollectionUuid,
+        seq_id: u64,
+    ) -> Result<(), SqlitePurgeLogsError> {
+        let topic = get_topic_name(&self.tenant_id, &self.topic_namespace, collection_id);
+
+        sqlx::query("DELETE FROM embeddings_queue WHERE topic = ? AND seq_id < ?")
```

Comment:
what do you think about getting the max_seq_id inside a transaction here instead of taking it as a method parameter?